plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/BomTest.java

```diff
@@ -16,85 +16,43 @@
 
 package com.google.cloud.tools.eclipse.appengine.libraries;
 
-import javax.xml.parsers.DocumentBuilder;
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
+import org.eclipse.core.runtime.CoreException;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.w3c.dom.DOMImplementation;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-import org.w3c.dom.Node;
 
 public class BomTest {
 
-  private Element dependencyManagement;
+  private Bom cloudBom;
   
   @Before
-  public void setUp() throws ParserConfigurationException {
-    DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-    factory.setNamespaceAware(true);
-    DocumentBuilder builder = factory.newDocumentBuilder();
-    DOMImplementation impl = builder.getDOMImplementation();
-
-    Document doc = impl.createDocument("http://maven.apache.org/POM/4.0.0", "project", null);
-
-    Node rootElement = doc.getDocumentElement();
-
-    dependencyManagement =
-        doc.createElementNS("http://maven.apache.org/POM/4.0.0", "dependencyManagement");
-    Element dependencies = doc.createElementNS("http://maven.apache.org/POM/4.0.0", "dependencies");
-
-    Element springDependency =
-        configureBom(doc, "org.springframework.boot", "spring-boot-dependencies", "2.0.0.RELEASE");
-    dependencies.appendChild(springDependency);
-    Element cloudDependency = configureBom(doc, "com.google.cloud", "google-cloud", "0.40.0-alpha");
-    dependencies.appendChild(cloudDependency);
-    dependencyManagement.appendChild(dependencies);
-    rootElement.appendChild(dependencyManagement);
-  }
-
-  private Element configureBom(Document doc, String groupId, String artifactId, String version) {
-    Element dependency = doc.createElementNS("http://maven.apache.org/POM/4.0.0", "dependency");
-    Element groupIdElement = doc.createElementNS("http://maven.apache.org/POM/4.0.0", "groupId");
-    Element artifactIdElement =
-        doc.createElementNS("http://maven.apache.org/POM/4.0.0", "artifactId");
-    Element versionElement = doc.createElementNS("http://maven.apache.org/POM/4.0.0", "version");
-
-    groupIdElement.setTextContent(groupId);
-    artifactIdElement.setTextContent(artifactId);
-    versionElement.setTextContent(version);
-
-    dependency.appendChild(groupIdElement);
-    dependency.appendChild(artifactIdElement);
-    dependency.appendChild(versionElement);
-    return dependency;
+  public void setUp() throws CoreException {
+    cloudBom = Bom.loadBom("com.google.cloud", "google-cloud", "0.41.0-alpha", null);
```

Comment:
Should this not be `com.google.cloud:google-cloud-bom`? (note trailing `-bom`)

Reply:
good catch. There is a google-cloud too but I don't know what that is.